xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -8,6 +8,7 @@
 
 #include "VideoPlayer.h"
 
+#include "application/ApplicationStackHelper.h"
```

Comment:
Making VideoPlayer dependent on Application is a nogo.

Reply:
I've moved it - are you happy with where it is now (in ApplicationPlayerCallback)??

Reply:
Yes, thanks.